tests/Fixtures/Integration/set/@PER-CS2.0.test-in.php

```diff
@@ -43,4 +43,8 @@ class Aaa implements
 
 $fn = fn ($a) => $a;
 
+$array = [
```

Comment:
pls add a test with array that is not multiline to confirm it's not touched

Reply:
Done